sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastHashJoinExecBase.scala

```diff
@@ -146,14 +157,40 @@ abstract class GpuBroadcastHashJoinExecBase(
 
     val rdd = streamedPlan.executeColumnar()
     val buildSchema = buildPlan.schema
+    val localIsNullAwareAntiJoin = isNullAwareAntiJoin
```

Comment:
Why we create a new variable? Could we just use `isNullAwareAntiJoin` and set `isNullAwareAntiJoin` with `val` qualifier?